apps/landing/components/changelog/changelog-grid-item.tsx

```diff
@@ -51,7 +50,7 @@ export async function ChangelogGridItem({ className, changelog }: Props) {
           value={`${baseUrl}/changelog#${changelog.tableOfContents.slug}`}
           className="mb-6 mt-12 2xl:pl-36 "
         >
-          <p className="">Copy Link</p>
+          <p className="">share on twitter</p>
```

Comment:
'Share on Twitter'

Could we use a tweet button here to share the URL? https://developer.twitter.com/en/docs/twitter-for-websites/tweet-button/overview?

Reply:
yes please!

Reply:
href={`https://twitter.com/intent/post?text=${BLOG_TITLE + OUR_URL}`}